spec/factories/site_statuses.rb

```diff
@@ -13,11 +13,15 @@
 
 FactoryBot.define do
   factory :site_status do
-    association(:course)
+    association :course, study_mode: :full_time_or_part_time
     association(:site)
     publish { 'N' }
     vac_status { :full_time_vacancies }
```

Comment:
This status isn't always going to be valid, why not do the `with_any_vacancy` logic by default?

Reply:
I think a static default value is simpler as it bypasses the `after(:build)` hook and is therefore less magic. Plus a bunch of stuff breaks if I make `with_any_vacancy` the default, so it didn't feel like the right time to switch this.